nautobot_ssot/filters.py

```diff
@@ -1,39 +1,35 @@
-"""Filtering logic for Sync and SyncLogEntry records."""
+"""Filtering for nautobot_ssot."""
 
-import django_filters
-from django.db.models import Q
-from nautobot.apps.filters import BaseFilterSet
+from nautobot.apps.filters import NautobotFilterSet, SearchFilter
 
-from .models import Sync, SyncLogEntry
+from nautobot_ssot import models
 
 
-class SyncFilterSet(BaseFilterSet):
-    """Filter capabilities for SyncOverview instances."""
+class SyncFilterSet(NautobotFilterSet):  # pylint: disable=too-many-ancestors
```

Comment:
Why the change here? From what I can see, NautobotFilterSet includes some additional mixins that'll potentially impact usage, ie CreatedUpdatedModelFilterSetMixin, RelationshipModelFilterSetMixin, and CustomFieldModelFilterSetMixin. I'm not entirely sure these mixins would be needed with the Sync object.